gradle/baseline.gradle

```diff
@@ -17,18 +17,37 @@ List<String> blacklistedBaselineProjects = [
         'profile-client-protobufs',
         'timestamp-impl']
 
+List<String> blacklistedErrorProneProjects = [
+        'atlasdb-console',
+        'atlasdb-dbkvs-hikari',
+        'atlasdb-jdbc',
+        'atlasdb-jdbc-tests',
+        'atlasdb-rocksdb',
+        'atlasdb-rocksdb-tests',
+        'atlasdb-service-server',
+        'cassandra-partitioner',
+        'commons-db',
+        'commons-executors',
+        'leader-election-impl',
+        'lock-impl',
+        'profile-client-protobufs',
+        'timestamp-impl']
+
 if (!blacklistedBaselineProjects.contains(project.name)) {
     apply plugin: 'com.palantir.baseline-checkstyle'
+} else if (!blacklistedErrorProneProjects.contains(project.name)) {
```

Comment:
This will *only* add error-prone for the projects that are in the first list, but not the second (looks like 3 projects to me). Is that intended?